x-pack/plugins/triggers_actions_ui/public/common/expression_items/of.tsx

```diff
@@ -108,7 +108,7 @@ export const OfExpression = ({
           />
         </ClosablePopoverTitle>
         <EuiFlexGroup>
-          <EuiFlexItem grow={false} className="watcherThresholdAlertAggFieldContainer">
+          <EuiFlexItem className="ofAggFieldContainer">
```

Comment:
You'll also need to use the `act` prefix for the Actions plugin.
```suggestion
          <EuiFlexItem className="actOf__aggFieldContainer">
```